build.gradle

```diff
@@ -81,11 +82,15 @@ configure(javaProjects) {
             java.srcDirs = ['src/main']
         }
     }
+
+    tasks.withType(GenerateModuleMetadata) {
```

Comment:
Added to resolve the exception `Signing Gradle Module Metadata is not supported for snapshot dependencies.`

Reply:
Nice